portfolio/src/main/webapp/piano.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Piano</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="piano.js"></script>
+  </head>
+<body>
+  <div id="content">
+
+    <p data-key="65" class = "key" data-note="C">C</p>
+    <p data-key="87" class = "key">C#</p>
+    <p data-key="83" class = "key">D</p>
+    <p data-key="69" class = "key">D#</p>
+    <p data-key="68" class = "key">E</p>
+    <p data-key="70" class = "key">F</p>
+    <p data-key="84" class = "key">F#</p>
+    <p data-key="71" class = "key">G</p>
+    <p data-key="89" class = "key">G#</p>
+    <p data-key="72" class = "key">A</p>
+    <p data-key="85" class = "key">A#</p>
+    <p data-key="74" class = "key">B</p>
+    <p data-key="75" class = "key">C</p>
+    <p data-key="79" class = "key">C#</p>
+    <p data-key="76" class = "key">D</p>
+    <p data-key="80" class = "key">D#</p>
+    <p data-key="186" class = "key">E</p>
+
+
+    <audio data-key="65" src="http://carolinegabriel.com/demo/js-keyboard/sounds/040.wav"></audio>
```

Comment:
It will be better to host these files on your website. Otherwise if these links break, your piano app will stop working. When doing this, you should make sure that you have permission to use/copy these files (or that they have a license that makes them free).
In this case we're taking them from a public tutorial but it's not clear what the licensing is. I found https://archive.org/details/opensource_audio has some clips under creative commons, otherwise there are other places where to get these sounds from.